server/controllers/componentsController.ts

```diff
@@ -122,11 +88,16 @@ export default ({
           [componentName]: viewModels[index],
         }
       },
-      { meta: userData },
+      {
+        meta:
+          user.authSource === 'nomis'
+            ? { caseLoads: user.caseLoads, activeCaseLoad: user.activeCaseLoad, services: user.services }
+            : DEFAULT_USER_ACCESS,
+      },
     )
   },
 })
 
 export type ComponentsData = Partial<Record<AvailableComponent, HeaderViewModel | FooterViewModel>> & {
-  meta: UserData
+  meta: PrisonUserAccess
```

Comment:
Renaming to try to make clear what sort of user data it is - case load and services - which I've named `PrisonUserAccess` since it's all stuff that Prison users can access.